image-layout.md

```diff
@@ -134,7 +134,7 @@ The `imageLayoutVersion` value will align with the OCI Image Specification versi
 
 ### oci-layout Example
 
-```json
+```json,title=OCI%20Layout&mediatype=oci%2Dlayout
```

Comment:
I'd [still][1] rather have “oci-layout file” (or similar) as the title and a real media type.

[1]: https://github.com/opencontainers/image-spec/pull/452#discussion_r102088422